src/analysisd/analysisd.c

```diff
@@ -1061,157 +1053,157 @@ RuleInfo *OS_CheckIfRuleMatch(Eventinfo *lf, RuleNode *curr_node)
      * weekday,
      * status,
      */
-    RuleInfo *currently_rule = curr_node->ruleinfo;
+    RuleInfo *rule = curr_node->ruleinfo;
```

Comment:
this change was caused by -Wshadow, because there is a global variable named like that in https://github.com/ossec/ossec-hids/blob/master/src/analysisd/rules.h#L169